clang/docs/ReleaseNotes.rst

```diff
@@ -872,6 +883,9 @@ Floating Point Support in Clang
   ``__builtin_exp10f128`` builtins.
 - Add ``__builtin_iszero``, ``__builtin_issignaling`` and
   ``__builtin_issubnormal``.
+- ``#pragma STDC CX_LIMITED_RANGE on-off-switch`` enables the naive mathematical
+  formulas for complex division and multiplication with no NaN checking of
+  results.
```

Comment:
Suggestion:

```
- Add support for C99's ``#pragma STDC CX_LIMITED_RANGE` feature.  This
  enables the naive mathematical formulas for complex multiplication and division,
  which are faster but do not correctly handle overflow and infinities.
```

I think we should add a `__has_feature` check for this and document it here.

Reply:
Yes.  Code should be able to check for whether the pragma is supported.